runner/conmon/options.go

```diff
@@ -62,6 +68,13 @@ func WithLogDriver(driver, path string) ConmonOption {
 	}
 }
 
+func WithLogDriverRaw(path string) ConmonOption {
+	return func(ci *ConmonInstance) error {
+		fullDriver := path
+		return ci.addArgs("--log-path", fullDriver)
```

Comment:
Why not just use `path` here? Or name the variable `fullDriver` above?

Reply:
Cuz I copied it and removed the if.  I could consolidate.

Honestly I'm not sure why the withlogdriver function needs driver and path - I'd think the unit tests would be closer to real life without having this intermediary deciding when we need a : and where it should go.  Unless the runner/conmon/ is used for something other than just unit testing.

Reply:
One day, I'd like it to replace the conmon-calling code in cri-o and podman, and give a nice go interface instead of the ad-hoc os/exec code that exists now.

changing it so that we do less validation on `runner/conmon` side is fine with me. I am not married to the implementation as is

Reply:
My main concern was I looked at the implementation and saw 
```
		if driver != "" {
			fullDriver = fmt.Sprintf("%s:%s", driver, path)
		}
```

So there's currently no way to test for ":filename".. Which, fine, it should default to k8s-file in that case.  So from a usage perspective, withLogDriver makes sense as driver,path...

Reply:
See changes below - that probably makes more sense short and long run.